conda_build/main_develop.py

```diff
@@ -32,6 +36,12 @@ def main():
         nargs='+',
         help="Path to the source directory."
     )
+    p.add_argument(
+                   '-l', '--post-link',
```

Comment:
aah - ok. So --no-pth-file would mean, don't append to conda.pth file. I can update.